src/ovp_pipeline/pulse.py

```
@@ -0,0 +1,107 @@
+"""Phase 37 — Pulse: poll-based tail of the JSONL event logs.
+
+Phases 32–36 already declared a closed ``event_type`` vocabulary in
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)